libs/util/src/lib/generate-leaderboard.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  AppFinishedPlayerValues,
+  AppLeaderboard,
+  AppLeaderboardEntry,
+  AppPlayerId,
+  AppPlayerLog,
+  AppPlayerLogId,
+  AppPlayerLogs,
+  AppPlayerStatus,
+  AppRaceId,
+  AppTimeoutPlayerValues,
+} from '@razor/models';
+import { extractId, extractIdType } from './extractIds';
+
+export const generateLeaderboard = (
```

Comment:
Can you please add short comments before each code stub, to describe what they do?

Reply:
Completed docs  in PR #14 under commit 4d6c57dec3d53ee4b0ea4ee596fa7368c3a0833d